package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "rsschool-codejam",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
```

Comment:
I recommend to add script for eslint here (maybe even with --fix flag)

Reply:
@anastasiaprivalova done